src/features/common/Layout/Navbar/index.tsx

```diff
@@ -124,7 +124,7 @@ export default function NavbarComponent(props: any) {
                   <div className={'link_icon'}>
                     <UserProfileIcon />
                   </div>
-                  <p className={router.pathname === SingleLink.onclick ? 'active_icon' : ''}>
+                  <p className={router.pathname === '/t/[id]' ? 'active_icon' : ''}>
```

Comment:
This highlights other profiles as well. 
Example if you visit my profile, Me will be highlighted

Reply:
Thank you for pointing this out. In that case I revert this change as otherwise it would be wrong. As the router.pathname is just the general path `t/[id]` I do not yet see any possibility to check if a user is currently viewing the "Me" area.